server/server_test.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"reflect"
+	"testing"
+	"time"
+
+	pb "chat_go/chat_protos"
+
+	st "chat_go/storage"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Tests 'GetUsers' method.
+func TestGetUsers(t *testing.T) {
+	assert := assert.New(t)
+	ctx := context.Background()
+	storageMock := &st.StorageMock{}
+	server := server{
+		storage: storageMock,
+	}
+
+	loginList := []string{"user1", "user2"}
+	fullnameList := []string{"u_user1", "u_user2"}
+
+	var storageUsers []st.User
+
+	for i := 0; i < len(loginList); i++ {
+		storageUsers = append(storageUsers, st.User{Login: loginList[i], FullName: fullnameList[i]})
+	}
+	storageMock.On("GetUsers").Return(storageUsers, nil)
+
+	var returnUsers []*pb.User
+
+	for i := 0; i < len(loginList); i++ {
+		returnUsers = append(returnUsers, &pb.User{Login: loginList[i], FullName: fullnameList[i]})
+	}
```

Comment:
This piece of code smells

Just define it in a common way

```suggestion
	storageUsers = []st.User{
	    {Login: "111", FullName: "u_111"},
	    {Login: "222", FullName: "u_222"},
	}
```

Reply:
Thanks, I've changed it